code/controllers/ExtensibleSearchAPI.php

```diff
@@ -93,22 +94,43 @@ public function toggleSuggestionApproved($request) {
 	 *	@URLparameter term <{SEARCH_TERM}> string
 	 *	@return JSON
 	 */
-
 	public function getSuggestions($request) {
 
-		if(Config::inst()->get('ExtensibleSearchSuggestion', 'enable_suggestions') && ($suggestions = $this->service->getSuggestions($request->getVar('term'), $request->getVar('page')))) {
+		if(Config::inst()->get('ExtensibleSearchSuggestion', 'enable_suggestions')) {
+			$suggestions = $this->service->getSuggestions($request->getVar('Search'), $request->getVar('page'));
```

Comment:
Yeah, I'd rather this didn't return a 404; the fact that the endpoint exists but with no results is a valid successful return for the http request. For which you _could_ use a 204 response, but 200 with an empty array is an acceptable alternative. 

From http://www.w3.org/Protocols/HTTP/HTRESP.html

> The 4xx codes are intended for cases in which the _client seems to have erred_

Now, if the client requested something that _should always have a result_ but nothing was found, that might be a different story...